src/controllers/EventsController.js

```diff
@@ -1,9 +1,177 @@
 // @flow
 
+import moment from 'moment';
 import Controller from './Controller';
+import CoreController from '../lib/CoreController';
+import route from '../decorators/route';
+import httpVerb from '../decorators/httpVerb';
+
+import v1Api from '../views/api_v1';
+import logger from '../lib/logger';
 
 class EventsController extends Controller {
+  _aliveInterval: ?string;
+  _lastMessage: ?Date;
+  _socket: ?Object;
```

Comment:
This won't work.  We actually open a socket for each user who requests to start listening.